uui/components/widgets/Avatar.tsx

```diff
@@ -1,4 +1,15 @@
-import { withMods } from '@epam/uui-core';
+import { devLogger, withMods } from '@epam/uui-core';
 import { Avatar as uuiAvatar, AvatarProps } from '@epam/uui-components';
 
-export const Avatar = withMods<AvatarProps>(uuiAvatar, () => []);
+export const Avatar = withMods<AvatarProps>(
+    uuiAvatar,
+    () => [],
+    (props) => {
+        if (__DEV__) {
```

Comment:
I propose moving this warning to the Avatar component in `@epam/uui-components'` package and make just re-export here, except one more wrapping with `withMods`